app/assets/stylesheets/application.scss

```diff
@@ -40,17 +47,113 @@ body {
   padding: 0 15px;
 }
 
-input[type=text].postfix.round,
-input[type=email].postfix.round {
-  border-radius: 1000px 0 0 1000px;
-  padding: 0;
+.background {
+  -webkit-background-size: contain;
+  -moz-background-size: contain;
+  -o-background-size: contain;
+  background: url(asset-path("footer-eight.jpg")) no-repeat;
+  background-size: contain;
+  background-position: center bottom;
+  display: flex;
+  flex-flow: column;
+  height: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 720px;
+  overflow: auto;
+}
+
+.frontpage-form {
+  input[type=text] {
+    background: transparent;
+    border: none;
+    border-radius: 1000px 0 0 1000px;
+    color: rgba(101, 247, 255, 0.75);
+    height: 60px;
+    line-height: 60px;
+    padding: 0;
+  }
+
+  input[type=submit], .button {
+    background-color: #000DBA;
+    border-color: #007095;
+    border-radius: 1000px;
+    color: #6FFDFF;
+    font-size: 1.8rem;
+    height: 60px;
+    line-height: 60px;
+    padding: 0;
+  }
+
+  .postfix {
+    font-size: 1rem;
+    height: 60px;
+    line-height: 60px;
+  }
+}
+
+.choice-row {
+  input[type=text] {
+    border: none;
+    border-radius: 0;
+    height: 60px;
+    line-height: 60px;
+    padding: 0;
+  }
+
+  .add-choice {
+    background-color: #000DBA;
+    border-color: #007095;
+    border-radius: 1000px 0 0 1000px;
+    color: #6FFDFF;
+    font-size: 1rem;
+    height: 60px;
+    line-height: 60px;
+    padding: 0;
+
+    .success {
+      &:hover,
+      &:active,
+      &:focus {
+        background-color: #25D3F0;
+      }
+    }
+  }
+
+  .remove-choice {
+    border-radius: 0 1000px 1000px 0;
+    padding: 0;
+    height: 60px;
+    line-height: 60px;
+  }
+}
+
+.form-floater {
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin-right: -50%;
+  transform: translate(-50%, -50%);
+  width: 45vw;
 }
 
 .init-choice-row {
   display: none;
+  .button.postfix.round.add-choice.success {
+    &:hover,
+    &:active,
+    &:focus {
+      background-color: #25D3F0;
```

Comment:
Color literals like `#25D3F0` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#25D3F0` should be written as `#25d3f0`